py/dml/messages.py

```diff
@@ -2392,3 +2392,7 @@ class PINT1(PortingMessage):
     types, then the value of the variable becomes 1, whereas for
     `int1` in DML 1.4 the value is -1."""
     fmt = "Change int1 to uint1"
+
+warnings = {name: cls for (name, cls) in globals().items()
```

Comment:
Just to double-check that I remember the semantics of `globals()` -- does it return the accessible vars _defined in the current module only_, or does it return accessible vars accessible at the top-level of the current module, no matter where they are defined?

Reply:
Somewhere inbetween, it seems -- globals() returns a dict that includes some builtin symbols like `__name__`,  `__loader__` and `__builtins__`, but not others like `__import__`.